blockchain/genesis.go

```diff
@@ -327,6 +327,10 @@ func (g *Genesis) ToBlock(baseStateRoot common.Hash, db database.DBManager) *typ
 	if g.BlockScore == nil {
 		head.BlockScore = params.GenesisBlockScore
 	}
+	// TODO-klaytn should set genesis block's base fee
```

Comment:
Not now, but it will be used later in `service chains` that want to use this policy from the beginning.
As you know, we didn't complete the homi configuration for service chain yet. It is a another issue.

Reply:
Got it. Thank you.